tests/java/lang/crlf_injection/testdata/main.java

```diff
@@ -0,0 +1,29 @@
+package inject;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.logging.*;
+
+public class CRLFInjection extends HttpServlet {
+  public void javaUtilLogging(HttpServletRequest req, HttpServletResponse res) {
+    String dangerous = req.getParameter("test");
+    String okay = "some known string";
+
+    logger = Logger.getLogger(Log.class);
+    logger.setLevel(Level.ALL);
+
+    // bearer:expected java_lang_crlf_injection
+    logger.info(dangerous);
+    // bearer:expected java_lang_crlf_injection
+    logger.info(dangerous.replace("hello", "world"));
+    // bearer:expected java_lang_crlf_injection
+    logger.info(dangerous.replace('\n', ""));
+    // bearer:expected java_lang_crlf_injection
+    logger.info(dangerous.replaceAll("\r", ""));
+
+    // okay
+    logger.config("hello world" + okay);
+    logger.info(dangerous.replace('\r', ' ').replace('\n', ' '));
+    logger.logrb(Level.INFO, safe, safe, dangerous + "bundle", safe);
```

Comment:
I'm not sure about this `+ "bundle"` and having it explicitly in the rule.

From the [doc](https://docs.oracle.com/javase%2F9%2Fdocs%2Fapi%2F%2F/java/util/logging/Logger.html) it looks like `logrb` accepts a `String` or a [`ResourceBundle`](https://docs.oracle.com/javase%2F9%2Fdocs%2Fapi%2F%2F/java/util/ResourceBundle.html)

Reply:
It seemed a bit arbitrary to me, to catch on the string. Shall we remove this case from the rule?

Reply:
I would write a test to catch `logrb` and deal with the bundle rather